### PR TITLE
Add support for ast.IfExp

### PR DESCRIPTION
**Pull request by kshxtij**
# Overview

Add support for `a if x else b` (ast.IfExp) statements.

# Details

Screenshot 2022-11-15 at 13 54 16

# References

- #91 

# Blocked by

NA

### CODE REVIEW

File: src/latexify/codegen/function_codegen.py [hunk 1]
```diff
@@ -517,6 +517,16 @@ def visit_If(self, node: ast.If) -> str:
         latex += self.visit(node)
         return latex + r", & \mathrm{otherwise} \end{array} \right."
 
+    def visit_IfExp(self, node: ast.IfExp) -> Any:
+        """Visit an ifexp node"""
+        latex = r"\left\{ \begin{array}{ll} "
+        cond_latex = self.visit(node.test)
+        true_latex = self.visit(node.body)
+        false_latex = self.visit(node.orelse)
```
Comment (odashi) [inline src/latexify/codegen/function_codegen.py:525]: Note: I can make some follow-up pull requests for this comment.

I think this line does not process the recursion. e.g.:

```python
if a:
    return x
elif b:
    return y
else:
    return z
```

and

```python
return x if a else (y if b else z)
```

should express the same logic, but this impl may generates different result.

**Review by odashi — COMMENTED** [review]
Hi, this is really good! Could you prepare some unit tests in `function_codegen_test.py` as well?

File: src/latexify/codegen/function_codegen.py (hunk 1 above)
Comment (kshxtij) [reply src/latexify/codegen/function_codegen.py:525]: Screenshot 2022-11-15 at 21 24 42

I was able to process the recursion you highlighted, but wanted to know if we need to support recursion within both code blocks of the `if  then  else` statement like shown in the 2nd example.

Comment (odashi) [reply src/latexify/codegen/function_codegen.py:525]: In `visit_If` the current implementation processes only `orelse` subtree. I think we could choose the same behavior at this point.

**Review by odashi — APPROVED** [review]

**Comment by odashi** [comment]
Great work, thanks!

**Comment by odashi** [comment]
Btw, CIs (Black and flake8) are still failing, I will merge this PR after resolving these issues.

**Pull request merged**